kythe/rust/indexer/tests/indexer_save_analysis.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Kythe Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate kythe_rust_indexer;
+use kythe_rust_indexer::indexer::save_analysis::load_analysis;
+
+extern crate runfiles;
+use runfiles::Runfiles;
+
+extern crate tempdir;
+use tempdir::TempDir;
+
+use std::fs;
+use std::path::PathBuf;
+
+#[test]
+fn load_analysis_works_properly() {
+    let temp_dir = TempDir::new("analysis_test").expect("Couldn't create temporary directory");
```

Comment:
I think the TempDir library should be avoided when building with bazel, as it may interact poorly with restricted build environments.  Instead use the test-specific TMP_TESTDIR, which bazel will give to each test that it runs.

Reply:
As an added benefit, you do not need to close the `temp_dir` once done with it, and if the test aborts, you don't leave temporary artifacts on the build machine.